core/server/models/base/index.js

```diff
@@ -355,14 +362,21 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
         }
 
         return itemCollection.fetchPage(options).then(function formatResponse(response) {
-            var data = {};
+            options.columns = allColumns;
+            var models = response.collection.toJSON(options);
 
             // re-add any computed properties that were stripped out before the call to fetchPage
-            options.columns = allColumns;
-            data[tableName] = response.collection.toJSON(options);
-            data.meta = {pagination: response.pagination};
+            // pick only requested before returning JSON
+            return Promise.map(models, function (model) {
```

Comment:
suggestion: 
no need to use `Promise.map`, here is no async operation happening. 
```
// pick only requested ...
models = _.map(models, function(model) {...});
```